src/RandomArt/randomTerms.js

```diff
@@ -0,0 +1,62 @@
+const randomTerms = [ "hep", 
+"no", 
+"col", 
+"cep", 
+"octopus", 
+"tel", 
+"mic", 
+"not", 
+"met", 
+"mod", 
+"tol", 
+"color", 
+"illustration", 
+"bull", 
+"lem", 
+"bot", 
+"blue", 
+"red", 
+"yellow", 
+"green", 
+"city", 
+"fef", 
+"son", 
+"ses", 
+"island", 
+"seven", 
+"bob", 
+"key", 
+"lef", 
+"mog", 
+"app", 
+"lif", 
+"ets", 
+"bug", 
+"ocean", 
+"mountain", 
+"lion", 
+"sketch", 
+"myth", 
+"glory", 
+"abstract", 
+"despair", 
+"hope", 
+"expression", 
+"impression", 
+"surreal", 
+"landscape", 
+"city", 
+"kil", 
+"lik", 
+"off", 
+"god", 
+"music", 
+"hands", 
+"night", 
+"dawn", 
+"jef", 
+"new", 
+"old",
+"tech" ]
```

Comment:
whoaza haha cant wait to see this play out on the display

Reply:
^^ This is a clever solution (and good call not storing it on the RandomArt component!)